oxd-server/src/main/java/org/gluu/oxd/server/Utils.java

```diff
@@ -116,4 +118,18 @@ public static Date addTimeToDate(Date date, int timeValue, int timeFormat) {
         cal.add(timeFormat, timeValue);
         return cal.getTime();
     }
+
+    public static Properties loadPropertiesFromFile(String filename, Properties props) {
+
+        try (FileInputStream fileistream = new FileInputStream(filename)) {
+            if (props == null) {
+                props = new Properties();
+            }
+            props.load(fileistream);
+            fileistream.close();
```

Comment:
@duttarnab this close is redundant, it will be closed automatically by try-with-resource statement.

Reply:
Done